docs/1.1-DRAFT/workflows.md

```diff
@@ -169,26 +175,70 @@ A workflow diagram may still be provided even if there is no programmatic `Softw
 
 ## Complying with BioSchemas Computational Workflow profile
 
-To comply with the BioSchemas [ComputationalWorkflow profile],
-where possible, data entities representing _workflows_ SHOULD describe these properties and their related contextual entities:
+Data entities representing _workflows_ (`@type: ComputationalWorkflow`)
+SHOULD comply with the
+BioSchemas [ComputationalWorkflow profile],
+where possible. When complying with this profile, the workflow data entities
+MUST describe these properties and their related contextual entities:
 
 * [name] giving a short descriptive human-readable name for the workflow
 * [programmingLanguage] identifying the workflow system, typed as `ProgrammingLanguage`
 * [creator] identifying the [Person](s) and/or [Organization](s) that assembled the workflow. (This may differ from the [author] of the conceptual workflow, e.g. if the workflow was converted from one language to another).
 * [dateCreated] the date the workflow was first created, e.g. `2020-05-23`
 * [license] identifying a [CreativeWork] that details the license for distributing or editing the workflow
 * [input] and [output] identifying contextual entities for the [FormalParameter] describing input and output parameters/variables that may be varied on different workflow executions
-* [sdPublisher] to identify the [Person] or [Organization] who has made the JSON-LD description of the workflow
-* [url] to identify a public webpage or source code repository for the workflow
+* [sdPublisher] to identify the [Person] or [Organization] who has made this JSON-LD description of the workflow
+* [url] to identify a public page or source code repository for the workflow
 * [version] to indicate the released version of this workflow file
 
-Contextual entities for [FormalParameter], referenced by `input` or `output`, SHOULD describe:
+The [ComputationalWorkflow] data entity SHOULD also describe, where relevant:
+* [citation] to identify a [CreativeWork] that should be cited for this workflow
+* [contributor] to identify [Person] or [Organization] that helped conceive the workflow
```

Comment:
What about the author? Contributor may have a little loose meaning and people often distinguish both

Reply:
I see that author is already included. Resolving conversation